internal/lpastore/update.go

```diff
@@ -46,27 +46,35 @@ func (c *Client) SendCertificateProvider(ctx context.Context, lpaUID string, cer
 	}
 
 	if certificateProvider.HomeAddress.Line1 != "" {
-		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/line1", New: certificateProvider.HomeAddress.Line1})
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/line1", New: certificateProvider.HomeAddress.Line1, Old: lpa.CertificateProvider.Address.Line1})
 	}
 
 	if certificateProvider.HomeAddress.Line2 != "" {
-		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/line2", New: certificateProvider.HomeAddress.Line2})
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/line2", New: certificateProvider.HomeAddress.Line2, Old: lpa.CertificateProvider.Address.Line2})
 	}
 
 	if certificateProvider.HomeAddress.Line3 != "" {
-		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/line3", New: certificateProvider.HomeAddress.Line3})
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/line3", New: certificateProvider.HomeAddress.Line3, Old: lpa.CertificateProvider.Address.Line3})
 	}
 
 	if certificateProvider.HomeAddress.TownOrCity != "" {
-		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/town", New: certificateProvider.HomeAddress.TownOrCity})
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/town", New: certificateProvider.HomeAddress.TownOrCity, Old: lpa.CertificateProvider.Address.TownOrCity})
 	}
 
 	if certificateProvider.HomeAddress.Postcode != "" {
-		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/postcode", New: certificateProvider.HomeAddress.Postcode})
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/postcode", New: certificateProvider.HomeAddress.Postcode, Old: lpa.CertificateProvider.Address.Postcode})
 	}
 
 	if certificateProvider.HomeAddress.Country != "" {
-		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/country", New: certificateProvider.HomeAddress.Country})
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/address/country", New: certificateProvider.HomeAddress.Country, Old: lpa.CertificateProvider.Address.Country})
+	}
+
+	if certificateProvider.Email != "" {
+		body.Changes = append(body.Changes, updateRequestChange{Key: "/certificateProvider/email", New: certificateProvider.Email, Old: lpa.CertificateProvider.Email})
+	}
+
+	if lpa.CertificateProvider.Channel != actor.ChannelOnline {
```

Comment:
I know we only have 2 channels, but I'd prefer `== channel.Paper` here as that is what you are saying it is changing from